utils/clients/tenderly.ts

```diff
@@ -267,13 +267,12 @@ async function simulateProposed(config: SimulationConfigProposed): Promise<Simul
     return getProposalId(log.args as unknown as ProposalEvent).eq(proposalId)
   })[0]
   if (!proposalCreatedEvent) throw new Error(`Proposal creation log for #${proposalId} not found in governor logs`)
-  const {
-    targets,
-    values,
-    signatures: sigs,
-    calldatas,
-    description,
-  } = proposalCreatedEvent.args as unknown as ProposalEvent
+  const { targets, signatures: sigs, calldatas, description } = proposalCreatedEvent.args as unknown as ProposalEvent
+
+  // Workaround an issue that ethers cannot decode the values properly.
+  // We know that the values are the 3rd parameter in
```

Comment:
Just noticed it should be the 4th value (index [3]) instead

Reply:
Ah thanks, fixed in https://github.com/Uniswap/governance-seatbelt/pull/94. Opened that PR to run CI, will merge once that completes if everything looks good